topics/metagenomics/tutorials/metagenomics-assembly/tutorial.md

```diff
@@ -0,0 +1,286 @@
+---
+layout: tutorial_hands_on
+
+title: "Metagenomic data assembly"
+zenodo_link: ""
+questions:
+  - "Why metagenomic data should be assembled?"
+  - "What is the difference between co-assembly and individual assembly?"
+  - "What is the difference between reads, contigs and scaffolds?"
+  - "How tools based on De Bruijn graph work?"  
+  - "How to assess the quality of metagenomic data assembly?"
+objectives:
+  - "Describe what an assembly is"
+  - "Describe what de-replication is"
+  - "Explain the difference between co-assembly and individual assembly"
+  - "Explain the difference between reads, contigs and scaffolds"
+  - "Explain how tools based on De Bruijn graph work"
+  - "Apply appropriate tools for analyzing the quality of metagenomic data"
+  - "Construct and apply simple assembly pipelines on short read data"
+  - "Apply appropriate tools for analyzing the quality of metagenomic assembly"
+  - "Evaluate the Quality of the Assembly with Quast, Bowtie2, and CoverM-Genome"
+time_estimation: "2H"
+key_points:
+  - "Assembly groups reads into contigs and scafolds."
+  - "De Brujin Graphs use k-mers to assembly reads"
+  - "MetaSPAdes and Megahit are assemblers"
+  - "Quast is the tool to assess the assembly quality"
+contributors:
+  - PlushZ
```

Comment:
Please also add yourself to the `CONTRIBUTORS.yaml` file

Reply:
I see myself there already, did I misunderstand?